requirements.txt

```diff
@@ -8,22 +8,21 @@
 # license agreement from NVIDIA CORPORATION is strictly prohibited.
 #
 
-configargparse~=0.15.2
+configargparse~=1.2.3
 Cython~=0.29.13
-h5py~=2.10.0
-numpy~=1.17.0
-pandas~=1.0.0
+h5py~=3.1.0
+numpy~=1.19.4
+pandas~=1.2.0
```

Comment:
no wheel package for Python3.6, https://pypi.org/project/pandas/1.2.0/#files
@ntadimeti can you also update the Readme file that we now support Python >= 3.7

Reply:
done!